### PR TITLE
Switch irc to libera.chat

### PR DESCRIPTION
**Pull request by jcvernaleo**
We haven't exactly decided to make the move yet, but I want this PR ready.  We have a new channel already and it does not look like freenode is a viable solution any more based on what has happened with many other channels.

### CODE REVIEW

**Comment by onyb** [comment]
I'm in favour of this switch, but a couple of points to note:
- The Freenode channel should still be around since that's where people from related communities like decred and lightning hang out.
- Some of us who are slightly more active on #btcd rely on a Matrix bridge for the scrollback. The Element team is reportedly working on a libera bridge, so maybe we should wait for that to be officially out before moving all conversations there.

**Comment by jcvernaleo** [comment]
I think the decred irc has been dead for a long time (ever since it moved to slack and then finally matrix).

I'm fine waiting for the matrix bridge to be done for a final move.

**Comment by jrick** [comment]
> I think the decred irc has been dead for a long time

it's dead, jim.

**Review by Rjected — APPROVED** [review]
I've been able to use Matrix/Element and chat on libera, so LGTM

File: README.md [hunk 1]
```diff
@@ -96,9 +96,8 @@ $ ./btcd
 
 ## IRC
 
-- irc.freenode.net
+- irc.libera.chat
 - channel #btcd
-- [webchat](https://webchat.freenode.net/?channels=btcd)
```
[the remark this answers is not in the document]
Comment (jcvernaleo) [reply README.md:101]: Good idea.  I wasn't aware of that (or it wasn't up yet when I did the PR initially).

Updated.

**Review by Roasbeef — APPROVED** [review]
LGTM 🍖

**Pull request merged**